README.md

```diff
@@ -147,6 +151,34 @@ You can also use a custom text, e.g.: `` [custom text](`MyModule.function/1`) ``
 Link to extra pages like this: `` [Up and running](Up and running.md) `` (skipping the directory
 the page is in), the final link will be automatically converted to `up-and-running.html`.
 
+## Admonition blocks
+
+You may want to draw attention to certain statements by taking them out of the
+content's flow and labeling them with a priority. These are called admonitions,
+sometimes are also known as asides or callouts. An admonition block is rendered
+based on the assigned label or class. `ex_doc` supports the following tags:
+`warning`, `error`, `info`, and `neutral` over header levels `h3` and `h4`.
+
+The syntax is as follows:
+
+```
+> #### Error {\: .error}
```

Comment:
@RobertDober Any advice on how to render this block as a raw piece?

```elixir
iex(1)> sample = """
...(1)> ```
...(1)> > #### Error {: .error}
...(1)> >
...(1)> > text
...(1)> ```
...(1)> """
"```\n> #### Error {: .error}\n>\n> text\n```\n"
iex(2)> EarmarkParser.as_ast sample
{:ok, [{"pre", [], [{"code", [], ["> #### Error \n>\n> text"], %{}}], %{}}], []}
iex(3)>
```

The parser seems to "eat" the `{: .error}` part. I also tried with `<pre>`.

Reply:
I tried it with `<pre>` and I think the issue is that earmark_parser is "eating" the new lines inside `<pre>`. I can take a look at it too:

```
<pre>
&gt; #### Error {: .error}
&gt;
&gt; This syntax will render an error block
</pre>
```

Reply:
> @RobertDober Any advice on how to render this block as a raw piece?
> 
> ```elixir
> iex(1)> sample = """
> ...(1)> ```
> ...(1)> > #### Error {: .error}
> ...(1)> >
> ...(1)> > text
> ...(1)> ```
> ...(1)> """
> "```\n> #### Error {: .error}\n>\n> text\n```\n"
> iex(2)> EarmarkParser.as_ast sample
> {:ok, [{"pre", [], [{"code", [], ["> #### Error \n>\n> text"], %{}}], %{}}], []}
> iex(3)>
> ```
> 
> The parser seems to "eat" the `{: .error}` part. I also tried with `\<pre\>`.

@milmazz  

Maybe as follows?

    iex(3)> markdown = """
    ...(3)> ~~~
    ...(3)> ```
    ...(3)> > #### Error{: .error}
    ...(3)> >
    ...(3)> > text
    ...(3)> ```
    ...(3)> ~~~
    ...(3)> """
    "~~~\n```\n> #### Error{: .error}\n>\n> text\n```\n~~~ \n"
    iex(4)> EarmarkParser.as_ast(markdown)
    {:ok,
     [{"pre", [], [{"code", [], ["```\n> #### Error\n>\n> text\n```"], %{}}], %{}}],
     []}

Reply:
still not what we want, I will open an issue, but I am afraid the fix needs to wait.

Do we agree that we want IAL markdown rendered verbatim in code blocks?

If you do not need syntax highlight you can use an indented code block

```
iex(13)> IO.puts markdown
    > #### Error {: .error}

    >
    > text

:ok
iex(14)> EarmarkParser.as_ast(markdown)
{:ok,
 [
   {"pre", [], [{"code", [], ["> #### Error {: .error}\n\n>\n> text"], %{}}],
    %{}}
 ], []}
```

Reply:
@josevalim @RobertDober pushed the workaround using an _indented code block_

re:

> Do we agree that we want IAL markdown rendered verbatim in code blocks?

IMO the IAL markdown syntax should be raw inside a fenced code block.